solution/ui/regulations/eregs-vite/src/components/SearchInput.vue

```diff
@@ -128,14 +128,16 @@ export default {
 
     methods: {
         submitForm() {
-            this.$emit("execute-search", { query: this.searchInputValue });
+            this.$emit("execute-search", {
+                query: encodeURIComponent(this.searchInputValue),
```

Comment:
text input string is encoded here; encoded string propagates through app for display in other components and to be included in API calls